model_analyzer/analyzer.py

```diff
@@ -165,6 +169,13 @@ def analyze(self, mode, quiet):
         if not quiet:
             self._result_manager.write_results()
 
+        top_3_model_config_names = self._get_top_3_model_config_names()
+        logger.info(f"To generate detailed reports for the "
```

Comment:
My expectation was that this would be logger.info(some_function_to_get_string()), and that the testing would just test the two get_string() functions.  That is really what we want to test and protect, not the lower level functions that function would use (although those get tested for free through this). You should be able to leverage most of the testing code you wrote.

Reply:
redesigned 👍